blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md

```diff
@@ -0,0 +1,196 @@
+cover: "./cover.jpg"
+description: "In this post we are presenting the achievements of the student projects during this year's edition of Google Summer of Code (GSOC)."
+author: "Niruandaleth"
+date: "2021-09-11"
+tags: ["TeraSpotlight"]
+---
+
+_We are proud to present the achievements of this year's student projects for the first half of the [Google Summer of Code]._
+_We are happy to announce that all five students passed their halftime evaluations._
+_The students will continue to contribute to different areas of our wide ecosystem, ranging from modernizing our web presence over world generation both, in space and on earth, to improvements in various gameplays._
+
+Eleven weeks of hard work are behind us, including the halftime and final evaluations.
+For the students, this means they have finished their projects and submitted their final work product.
+In addition, the students introduced their achievements to our community in a final "GSoC Show & Tell" session.
+
+Read on to learn what the students achieved in their projects!
+
+<div class="col" markdown="1">
+ <div class="card w-100 mt-4 shadow p-3 mb-5 bg-white rounded" style="height: 180px">
+   <div class="row">
+   <div class="col-2"> 
+     <img  class="rounded-circle ml-4" style="height: 30%" src="https://avatars.githubusercontent.com/ahv15" />
+   </div>
+   <div class="col-9"> 
+     <p class="card-title text-muted">Alluri Harshit Varma</p>
+    <h5 class="card-text">Quality Improvements for Light & Shadow</h5>
+    <hr />
+    <div>
+   <a class="text-warning" href="https://summerofcode.withgoogle.com/organizations/5338575677161472/#4719812875386880">PROJECT CARD</a>
+    </div>
+   </div>
+   </div>
+</div>
+
+  Alluri (aka _@ahv15_) overhauled the [`LightAndShadow`](https://github.com/Terasology/lightandshadow) module including removal of unused code and refactorings towards proper event-based Entity-Component-System architecture.
+  Further, Alluri fixed a lot of gameplay bugs related to world generation, starting and respawning inventory, weapon drops and pickups, et cetera.
+  In terms of features, the student introduced a pre-game phase designed to balance delays in team setup both with regards to players joining and teams discussing strategy.
+  This pre-game phase also gives players time to buy in the new shop screen that was extracted from our tower-defense gameplay `GooeyDefence`.
+  Accessible throughout all game stages is the new statistics screen that shows the player distribution across the teams as well as stats on their deaths, kills and scored flag points.
+  The final feature, active defensive structures to be placed by players to affect players of the opposing team is still pending in the final review cycles.
+
+  <div align="center">
+    <img src="./las.jpg" width="60%" />
+    <figcaption>Light & Shadow - Shop Screen, Pre-game Phase with separating Domes and Team Balancing, Statistics Screen</figcaption>
+  </div>
```

Comment:
The caption is rendered quite small, maybe the CSS is not picked up properly? 

![image](https://user-images.githubusercontent.com/1448874/211119737-98294cff-6fd7-4453-a89d-7349141c955c.png)

Reply:
should be fixed now